app/project.properties

```diff
@@ -11,5 +11,5 @@
 split.density=false
 # Project target.
 target=Google Inc.:Google APIs:19
-android.library.reference.1=../libraries/mapballoons/android-mapviewballoons
```

Comment:
Something definitely needed to be updated, but it was probably the 3 => 2, I imagine I just have some eclipse checkbox checked somewhere that caused the separator change. I'm happy to change it, is there any reason to prefer one style? (i'm guessing the forward slashes, since they were already there)